### PR TITLE
Use GitHub Workflows

### PR DESCRIPTION
**Pull request by woody-apple**
#### Problem
IPv6 Testing has been challenging, as well as keeping all our tests running in our mixed IPv6 IPv4 world. This seems to align all the testing run at once.

 #### Summary of Changes
* Move to GitHub Workflows, enable IPv6 testing by default
* Disabled bloat check for now @andy31415 offered to fix this

### CODE REVIEW

File: scripts/examples/esp_echo_app.sh [hunk 1]
```diff
@@ -19,6 +19,9 @@
 set -x
 env
 
+ls -la /var/
```
Comment (andy31415) [inline scripts/examples/esp_echo_app.sh:22]: these shoud go away?

**Review by andy31415 — COMMENTED** [review]
I see changes in third_party/openthread/repo

Guessing not intended?

File: scripts/examples/esp_echo_app.sh (hunk 1 above)
Comment (woody-apple) [reply scripts/examples/esp_echo_app.sh:22]: I'll remove that, thanks!

Comment (woody-apple) [reply scripts/examples/esp_echo_app.sh]: done

**Review by saurabhst — APPROVED** [review]

**Comment by woody-apple** [comment]
> I see changes in third_party/openthread/repo
> 
> Guessing not intended?

Fixed!

**Comment by woody-apple** [comment]
@jelderton @hawk248 ?

**Review by BroderickCarlin — APPROVED** [review]

**Review by anush-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
@andy31415 ?

**Review by andy31415 — APPROVED** [review]

**Pull request merged**